borg/helpers.py

```diff
@@ -235,16 +235,21 @@ def parse_timestamp(timestamp):
         return datetime.strptime(timestamp, '%Y-%m-%dT%H:%M:%S').replace(tzinfo=timezone.utc)
 
 
+def load_excludes(fh):
```

Comment:
maybe rather use f or file or exclude_file instead of fh.

fh is a very popular variable name meaning file handle in the C / OS sense (that is NOT a python file object, but an integer). as python has file objects as well as integer file handles, using fh and meaning a file object is misleading.